labcontrol/db/process.py

```diff
@@ -3819,3 +3826,407 @@ def _generate_amplicon_prep_information(self):
+        Gathers metadata used by above method and performs initial munging
+        for clarity.
+
+        Returns
+        -------
+        list: dict, each one representing a row of results.
+
+        #note: this fetchall() seemed appropriate, as we only expect to return
+        #several hundred results at most. This allows us to capture the results
+        #and clean them up before handing them off. This also allows us to
+        #refactor this query in time without touching the rest of the code.
+        """
+        inst_mdl, equipment, reagent = self._get_additional_prep_metadata()
+
+        sql = """
+                SELECT
+                    study.study_id,
+                    study_sample.sample_id,
+                    study.study_alias AS project_name,
+                    study_sample.sample_id AS orig_name,
+                    study.study_description AS experiment_design_description,
+                    samplewell.row_num AS row_num,
+                    samplewell.col_num AS col_num,
+                    samplecp.content,
+                    sampleplate.external_id AS sample_plate,
+                    platingprpr.run_personnel_id AS plating,
+                    gdnaextractpr.extraction_kit_id,
+                    gdnaextractpr.epmotion_robot_id AS gepmotion_robot_id,
+                    gdnaextractpr.epmotion_tool_id,
+                    gdnaextractpr.kingfisher_robot_id,
+                    libpreppr.kappa_hyper_plus_kit_id,
+                    libpreppr.stub_lot_id,
+                    libpreppr.normalization_process_id,
+                    primersetcp.barcode_seq AS barcode_i5,
+                    primersetcp2.barcode_seq AS barcode_i7,
+                    primersetcp.primer_set_id AS primer_set_id_i5,
+                    primersetcp2.primer_set_id AS primer_set_id_i7,
+                    primersetcp.external_id AS i5_index_id,
+                    primersetcp2.external_id AS i7_index_id,
+                    primersetplate.external_id AS primer_plate_i5,
+                    primersetplate2.external_id AS primer_plate_i7,
+                    primerworkingplateprpr.run_date AS primer_date_i5,
+                    primerworkingplateprpr2.run_date AS primer_date_i7
+                FROM labman.plate libprepplate
+                LEFT JOIN labman.well libprepwell ON (
+                    libprepplate.plate_id = libprepwell.plate_id)
+                LEFT JOIN labman.composition libprepcpcp ON (
+                    libprepwell.container_id = libprepcpcp.container_id)
+                LEFT JOIN labman.library_prep_shotgun_process libpreppr ON (
+                    libprepcpcp.upstream_process_id = libpreppr.process_id)
+                LEFT JOIN labman.library_prep_shotgun_composition libprepcp ON
+                    (libprepcpcp.composition_id = libprepcp.composition_id)
+                LEFT JOIN labman.normalized_gdna_composition b ON (
```

Comment:
Please rename the "b" and "c" aliases to normgdnacp and compgdnacp to match the naming conventions of this sql statement.